elide-integration-tests/src/test/java/com/yahoo/elide/fieldLevelTest/FieldLevelParentClassIdIT.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2017, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.fieldLevelTest;
+
+import com.yahoo.elide.initialization.AbstractIntegrationTestInitializer;
+import com.yahoo.elide.initialization.AssignedIdLongIntegrationTestApplicationResourceConfig;
+import com.yahoo.elide.utils.JsonParser;
+import org.apache.http.HttpStatus;
+import org.testng.annotations.Test;
+
+import static com.jayway.restassured.RestAssured.given;
+
+public class FieldLevelParentClassIdIT extends AbstractIntegrationTestInitializer {
+    private final JsonParser jsonParser = new JsonParser();
+
+    private static final String JSONAPI_CONTENT_TYPE = "application/vnd.api+json";
+
+    public FieldLevelParentClassIdIT() {
```

Comment:
You can remove this constructor (rely on the default constructor).  Then you should be able to remove some of the initialization scaffolding classes below.